pkg/output/json.go

```diff
@@ -52,12 +59,41 @@ func newResourceJSON(r *schema.Resource) resourceJSON {
 	}
 }
 
-func ToJSON(resources []*schema.Resource) ([]byte, error) {
+func showSkippedResourcesJSON(resources []*schema.Resource, showDetails bool) []string {
+	unSupportedTypeCount, _, unSupportedCount, _ := terraform.CountSkippedResources(resources)
+	if unSupportedCount == 0 {
+		return nil
+	}
+	message := fmt.Sprintf("\n%d out of %d resources couldn't be estimated as Infracost doesn't support them yet (https://www.infracost.io/docs/supported_resources)", unSupportedCount, len(resources))
```

Comment:
Do we need a \n at the start here?

Reply:
In the table output it becomes hard to distinguish the aggrgated prices (last line of output) from this output but you are right I will remove it for now.

Reply:
Done